.github/workflows/ci.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
             ROS_DISTRO: jazzy
     env:
       CXXFLAGS: >-
-        -Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls
+        -Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unknown-warning-option
```

Comment:
I saw this too in another PR and simply rerunning the check somehow fixed it? Either way, should be a separate PR.

Reply:
#2967